Create ui-c8y-1021-22-81-branding-should-be-able-to-find-public-options-app-on-management.md (#3320)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-22-81-branding-should-be-able-to-find-public-options-app-on-management.md

---
date: ""
title: branding should be able to find public options app on management tenant (#9091)
product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63601
version: 1021.22.81
---

> branding should be able to find public options app on management tenant (#9091)

Reviewer: ```suggestion
Previously, the branding manager was in certain cases unable to locate the public-options application on the management tenant, preventing the user from custimizing the branding.
This has now been resolved.
```

---
date: ""

> title: Branding can now find public options app on management tenant

Reviewer 2: ```suggestion
title: Resolved issues with branding manager which prevented customizing the branding
```

Reviewer 2: @reey This is a perfect example of a title which does not describe the issue from the user perspective but from the developer perspective. Just FYI :-).

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63601
version: 1021.22.81
---

> Previously, the branding manager was in certain cases unable to locate the public-options application on the management tenant, preventing the user from custimizing the branding.

Reviewer 2: ```suggestion
Previously, the branding manager was in certain cases unable to locate the public-options application on the {{< management-tenant >}}, preventing the user from customizing the branding.
```

> This has now been resolved.

Reviewer 2: ```suggestion
This issue has now been resolved.
```
